core/common/assets/js/api/core/store.js

```diff
@@ -31,6 +31,19 @@ export default class Store {
 			// Use an empty function instead of empty object since an empty object
 			// isn't a valid reducer value.
 			reducer: () => {},
+			middleware: ( getDefaultMiddleware ) => getDefaultMiddleware( {
```

Comment:
Why not move it to QUnit?

Reply:
For some reason the `bootstrap` class approach didn't work when I tried.